src/rpc/rawtransaction_util.cpp

```diff
@@ -118,13 +118,13 @@ CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniVal
                 throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid DigiByte address: ") + name_);
             }
 
-            if (!destinations.insert(destination).second) {
-                throw JSONRPCError(RPC_INVALID_PARAMETER, std::string("Invalid parameter, duplicated address: ") + name_);
-            }
-
             CScript scriptPubKey = GetScriptForDestination(destination);
             CAmount nAmount = AmountFromValue(outputs[name_]);
 
+            if ((nAmount!=600) && (!destinations.insert(destination).second)) { //if not a DigiAsset only allow 1 output per asset
```

Comment:
I am very concerned about the potential risks this introduces by hacking this condition in an RPC method that was never designed for this use case:

**Possible Risks:**

1. **Code Complexity Increase:** The added condition introduces complexity to the transaction construction logic, making it harder to maintain and debug.

2. **Potential for Errors:**
   - **Edge Cases:** There could be edge cases where a transaction might not be handled correctly, especially if future changes are made without considering this unique condition.
   - **Incorrect Amounts:** If the amount is mistakenly set to 600 for non-DigiAssets, it might bypass the duplicate check.

3. **Security Concerns:**
   - **Validation Issues:** The condition might introduce validation issues, potentially allowing malformed or fraudulent transactions.

4. **Compatibility:**
   - **Backward Compatibility:** The change assumes that DigiAssets are always associated with 600, which may not hold true universally or in future updates.
   
   I can't support this change, but I could consider a dedicated set of RPC methods and utilities for DigiAssets.

Reply:
Any proposed new RPC methods and utilities must also come with unit and functional tests.

Reply:
> Any proposed new RPC methods and utilities must also come with unit and functional tests.

No new method are being proposed.  But if you want tests specific to this that is fine.  I just want clear answer to the following question.

Do you want all outputs to be allowed to go to the same address no matter the amount or do you want a limited exception?

Reply:
I do not support changing this RPC utility under any circumstances, given the risks I stated above. What I would favor reviewing is a dedicated RPC method (or methods) and utilities designed explicitly for DigiAssets that make no modifications whatsoever to the core protocol or any existing RPC methods.

The proposed change needs to be revised so that this condition is not hard coded in any existing RPC methods.